File: be-api/src/main/java/cd/beapi/dto/response/PatientResponse.java
package cd.beapi.dto.response;

import java.time.Instant;
import java.time.LocalDate;

public record PatientResponse(
        Long id,
        String code,
        String fullName,
        String phone,
        LocalDate dob,
        Boolean gender,
        String address,
        String guardianName,
        String guardianPhone,
        Long version,
        Instant createdAt,
        Instant modifiedAt
) {}

File: fe-admin/src/app/core/model/response/patient-response.ts
export interface PatientResponse{
    id:number;
    code:string;
    fullName:string;
    phone:string;
    dob:Date;
    gender:boolean;
    address:string;
    version:number
    guardianName:string;
    guardianPhone:string;
    createdAt:Date;
    modifiedAt:Date;
}